ee/consoleuser/consoleuser_windows.go

```diff
@@ -49,7 +68,8 @@ func CurrentUids(ctx context.Context) ([]string, error) {
 		}
 
 		// Check for a couple well-known accounts that we know we don't want to create desktop processes for.
-		if shouldIgnoreSession(sessionData) {
+		sessionUsername := usernameFromSessionData(sessionData)
+		if shouldIgnoreSession(sessionData, sessionUsername) {
```

Comment:
We should make sure this gets logged.

Reply:
We'll have the `no user explorer process found for <username>` logs from the runner!

Reply:
True... Though we need to infer that it's why we didn't start a desktop

Reply:
And I guess I really _don't_ want a log every 5s